src/System.Net.Security/src/System/Net/Security/SslStream.cs

```diff
@@ -525,21 +525,8 @@ public override void EndWrite(IAsyncResult asyncResult)
         // Returns:
         // 
         //     A Task<int> representing the read.
-        public override Task<int> ReadAsync(byte[] buffer, int offset, int size, CancellationToken cancellationToken)
-        {
-            if (cancellationToken.IsCancellationRequested)
-            {
-                return Task.FromCanceled<int>(cancellationToken);
-            }
-
-            return Task.Factory.FromAsync(
-                (bufferArg, offsetArg, sizeArg, callback, state) => ((SslStream)state).BeginRead(bufferArg, offsetArg, sizeArg, callback, state),
-                iar => ((SslStream)iar.AsyncState).EndRead(iar),
-                buffer,
-                offset,
-                size,
-                this);
-        }
+        public override Task<int> ReadAsync(byte[] buffer, int offset, int size, CancellationToken cancellationToken) =>
```

Comment:
> Assuming we have special logic somewhere that changes the behavior based on thing being overridden?

Are you asking why I didn't just delete the method entirely?  I was leaving it more because a) overrides can affect the contract, and b) I'm hopeful we'll actually put real logic back here in the not-too-distant-future, per some of my comments in the PR description.